e2e/test/scenarios/actions/actions-reproductions.cy.spec.js

```diff
@@ -404,3 +404,37 @@ describe("issue 51020", () => {
     });
   });
 });
+
+describe("issue 51282", { tags: ["@external", "@actions"] }, () => {
+  beforeEach(() => {
+    H.restore("postgres-12");
+    cy.signInAsAdmin();
+    H.setActionsEnabledForDB(WRITABLE_DB_ID);
+    H.createModelFromTableName({
+      tableName: "orders",
+      modelName: "Order",
+      idAlias: "modelId",
+    });
+    cy.get("@modelId").then(modelId => {
+      H.visitModel(modelId);
+    });
+  });
+
+  it("should disable 'make public' switch in new action modal and show an explanatory tooltip (metabase#51282)", () => {
```

Comment:
I think a unit test would be fine as well

Reply:
ab7043056b9a001b9823df56e5a9ee33f9173de0 + 6de77ba21153acc87743c9bb4167af91acdac852